gce/worker/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN pip install pyasn1 click urllib3 --upgrade
 
 RUN pip install git+https://github.com/zarr-developers/zarr \
                 git+https://github.com/pydata/xarray \
-                git+https://github.com/dask/gcsfs@f99177b31c44fcc404619b2876a77cdcda955a75 \
+                git+https://github.com/dask/gcsfs \
```

Comment:
Same here.  We should probably pin to something.  

@martindurant has there been a release since this commit?  It looks like it was committed on March 22nd

Reply:
Yes, v0.1.0 on April 20